src/media/js/views/app.js

```diff
@@ -75,10 +67,6 @@ define('views/app',
                 }
             });
 
-            if (!sync) {
-                return;
-            }
-
             if (app) {
                 tracking.setPageVar(6, 'App name', app.name, 3);
```

Comment:
We may want a mapping of vars to their use at some point just so we have a single point of reference for which values are used. Maybe something like `tracking.dimensions.appName`. We get tracking questions often enough and it's easy to forget.